src/main/java/bitbucketpullrequestbuilder/bitbucketpullrequestbuilder/BitbucketRepository.java

```diff
@@ -121,6 +120,10 @@ public void init(ApiClient client) {
                 targetPullRequests.add(pullRequest);
             }
         }
+
+        if (trigger.getBuildChronologically()){
```

Comment:
Since `buildCronologically` is always set to true, the `targetPullRequests` is always reversed, which means the default operation is changed behavior.

Reply:
Yes, if user set value for this parameter to `true` - it will be applied to all PRs that are should be built by current pipeline.